ARC specification.md

```diff
@@ -186,21 +186,29 @@ Notes:
 
 Workflow execution and metadata MUST be described using the [Common Workflow Language](https://www.commonwl.org/) (CWL), [v1.2](https://www.commonwl.org/v1.2/) or higher, in a file `workflow.cwl`, which MUST be placed in the subdirectory containing all files specific to this workflow under the top-level `workflows` subdirectory. This file MUST contain either of:
```

Comment:
add a file tree here as example

Reply:
I updated the example file structure and linked it there